numpy/polynomial/tests/test_legendre.py

```diff
@@ -203,6 +203,9 @@ def test_legint(self):
         assert_raises(ValueError, leg.legint, [0], .5)
         assert_raises(ValueError, leg.legint, [0], -1)
         assert_raises(ValueError, leg.legint, [0], 1, [0, 0])
+        assert_raises(ValueError, leg.legint, [0], lbnd=[0])
+        assert_raises(ValueError, leg.legint, [0], scl=[0])
+        assert_raises(ValueError, leg.legint, [0], axis=.5)
```

Comment:
Shouldn't this raise a `TypeError`?

Reply:
Hmm. I was just following the documentation ... It would probably be more correct if all the errors checked here were Type errors except possibly the third and sixth.